src/CommunityVoices/App/Website/View/Image.php

```diff
@@ -237,12 +237,7 @@ public function postImageUpload($request)
 
     public function postImageUpdate($request)
     {
-        $response = new HttpFoundation\RedirectResponse(
-            $request->headers->get('referer')
-        );
 
-        $this->finalize($response);
-        return $response;
```

Comment:
Uh oh!  We need to return at least something, I figure?

Reply:
Maybe I should delete this function entirely on the website layer since it is treated as an AJAX Request?

Reply:
That makes sense, you could use the `use-api` parameter in `Routes.json` to make a route only ever go to the api.

Reply:
Just make sure the website version is for sure never used.

Reply:
use-api: true was already on routes.json, just checked

Reply:
Feel free to remove it entirely then.

Reply:
I am a little confused by what I was saying above, postImageUnpair is used on the website, not just the API. This was the pull request I made to fix the issue with imageUnpairing. Resolving this conversation since I don't think this is relevant any longer